test/journeys/lib/widgetPages/base.js

```diff
@@ -0,0 +1,108 @@
+import uuid from 'uuid';
+
+export default class BaseWidgetObject {
+  get hasWidgetContainer() { return this.browser.isVisible(this.elements.widgetContainer); }
+
+  get hasWidget() { return this.browser.isVisible(this.elements.widgetBody); }
+
+  constructor({aBrowser, user}) {
```

Comment:
Some jsdoc would be good here. Defining what all is in the user object would be helpful.

Reply:
Good call.